### PR TITLE
Fix too restrictive body argument handling

### PR DESCRIPTION
**Pull request by toopy**

### CODE REVIEW

**Comment by toopy** [comment]
@k4nar @tbcvl .. another one for the win ?

File: mock_services/rules.py [hunk 1]
```diff
@@ -83,7 +85,8 @@ def update_rest_rules(rules, content_type='application/json'):
             raise NotImplementedError('invalid method "{method}" for: {url}'.format(**kw))  # noqa
 
         # set callback if does not has one
-        if 'text' not in kw:
+        provided = [x for x in _BODY_ARGS if x in kw]
+        if not provided:
```
Comment (k4nar) [inline mock_services/rules.py]: if not any(x for x in _BODY_ARGS if x in kw):

**Comment by toopy** [comment]
UP^

**Comment by toopy** [comment]
OPR ?

**Comment by k4nar** [comment]
OPR

**Comment by toopy** [comment]
OPM^

**Comment by k4nar** [comment]
:+1:

**Pull request merged**